App/Course App/SharedUI/DesignSystem.swift

```diff
@@ -0,0 +1,77 @@
+// swiftlint:disable:this file_name
+//  DesignSystem.swift
+//  Course App
+//
+//  Created by Tomáš Duchoslav on 22.05.2024.
+//
+
+import SwiftUI
+import UIKit
+
+struct TextTypeModifier: ViewModifier {
```

Comment:
I would separate those extension into own files. In common project it would grow quite fast. Also having the definition of design items and view modifier in one file is not the best, there may come multiple other modifiers using the same pieces.